# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Bruno Queiros <bqueiros@gmail.com>, 2013
# João Diogo Ferreira <jdiogo2003@netcabo.pt>, 2009
# Rui Gouveia <rui.gouveia@gmail.com>, 2010, 2012
# Vit Mojzis <vmojzis@redhat.com>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-11-20 17:16+0100\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2017-09-01 08:32-0400\n"
"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
"Language-Team: Portuguese (http://www.transifex.com/projects/p/fedora/"
"language/pt/)\n"
"Language: pt\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Zanata 3.9.6\n"

#: ../src/allow_anon_write.py:27
msgid ""
"\n"
"    SELinux policy is preventing an httpd script from writing to a public\n"
"    directory.\n"
"    "
msgstr ""
"\n"
"    A política do SELinux impediu um 'script' de HTTP de gravar numa\n"
"    pasta pública.\n"
"    "

#: ../src/allow_anon_write.py:32
msgid ""
"\n"
"    SELinux policy is preventing an httpd script from writing to a public\n"
"    directory.  If httpd is not setup to write to public directories, this\n"
"    could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    A política do SELinux está a impedir um script http de gravar numa pasta\n"
"    pública. Se o http não estiver configurado para escrever em pastas\n"
"    públicas, isto poderá assinalar uma tentativa de intrusão.\n"
"    "

#: ../src/allow_anon_write.py:38
msgid ""
"\n"
"    If httpd scripts should be allowed to write to public directories you "
"need to turn on the $BOOLEAN boolean and change the file context of the "
"public directory to public_content_rw_t.  Read the httpd_selinux\n"
"    man page for further information:\n"
"    \"setsebool -P $BOOLEAN=1; chcon -t public_content_rw_t <path>\"\n"
"    You must also change the default file context labeling files on the "
"system in order to preserve public directory labeling even on a full relabel."
"  \"semanage fcontext -a -t public_content_rw_t <path>\"\n"
"    "
msgstr ""

#: ../src/allow_anon_write.py:44
msgid ""
"If you want to allow $SOURCE_PATH to be able to write to shared public "
"content"
msgstr ""

#: ../src/allow_anon_write.py:45
msgid ""
"you need to change the label on $TARGET_PATH to public_content_rw_t, and "
"potentially turn on the allow_httpd_sys_script_anon_write boolean."
msgstr ""

#: ../src/allow_execheap.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from changing the access\n"
"    protection of memory on the heap.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir que o $SOURCE_PATH mude a protecção de acesso\n"
"    à memória na 'heap' (área de dados).\n"
"    "

#: ../src/allow_execheap.py:32
msgid ""
"\n"
"    The $SOURCE application attempted to change the access protection of "
"memory on\n"
"    the heap (e.g., allocated using malloc).  This is a potential security\n"
"    problem.  Applications should not be doing this. Applications are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  If $SOURCE does not "
"work and\n"
"    you need it to work, you can configure SELinux temporarily to allow\n"
"    this access until the application is fixed. Please file a bug\n"
"    report against this package.\n"
"    "
msgstr ""
"\n"
"    A aplicação $SOURCE tentou mudar a protecção do acesso à memória\n"
"    na 'heap' (a memória reservada com o 'malloc'). Este é um problema\n"
"    potencial de segurança. As aplicações não deverão fazer isto. As "
"aplicações\n"
"    são algumas vezes programadas incorrectamente e pedem esta\n"
"    permissão. A página Web acerca dos\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">Testes de "
"Protecção de Memória do SELinux</a>\n"
"    explica como remover este requisito. Se o $SOURCE não funcionar\n"
"    e você precisar dele para funcionar, poderá configurar o SELinux\n"
"    temporariamente para permitir este acesso, pelo menos até a aplicação\n"
"    estar corrigida. Envie por favor um relatório\n"
"    de erros para este pacote.\n"
"    "

#: ../src/allow_execheap.py:44
msgid ""
"\n"
"    If you want $SOURCE to continue, you must turn on the\n"
"    $BOOLEAN boolean.  Note: This boolean will affect all applications\n"
"    on the system.\n"
"    "
msgstr ""
"\n"
"    Se quiser que o $SOURCE continue, deverá activar o booleano\n"
"    $BOOLEAN. Nota:  Este booleano irá afectar todas as aplicações\n"
"    no sistema.\n"
"    "

#: ../src/allow_execheap.py:50
msgid ""
"If you do not think $SOURCE_PATH should need to map heap memory that is both "
"writable and executable."
msgstr ""

#: ../src/allow_execheap.py:51
msgid "you need to report a bug. This is a potentially dangerous access."
msgstr "precisa de reportar um erro. Este acesso é potencialmente perigoso."

#: ../src/allow_execheap.py:52 ../src/allow_execstack.py:91
#: ../src/kernel_modules.py:46 ../src/mmap_zero.py:48
#: ../src/selinuxpolicy.py:46 ../src/setenforce.py:29 ../src/setenforce.py:50
#: ../src/sys_module.py:30 ../src/sys_module.py:48
msgid "Contact your security administrator and report this issue."
msgstr "Contacte o administrador de segurança e reporte este problema."

#: ../src/allow_execmod.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from loading $TARGET_PATH which "
"requires text relocation.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir que o $SOURCE_PATH carregue o $TARGET_PATH, que "
"precisa de recolocação de texto.\n"
"    "

#: ../src/allow_execmod.py:33
msgid ""
"\n"
"    The $SOURCE application attempted to load $TARGET_PATH which\n"
"    requires text relocation.  This is a potential security problem.\n"
"    Most libraries do not need this permission. Libraries are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  You can configure\n"
"    SELinux temporarily to allow $TARGET_PATH to use relocation as a\n"
"    workaround, until the library is fixed. Please file a\n"
"bug report.\n"
"    "
msgstr ""

#: ../src/allow_execmod.py:45
msgid ""
"\n"
"    The $SOURCE application attempted to load $TARGET_PATH which\n"
"    requires text relocation.  This is a potential security problem.\n"
"    Most libraries should not need this permission.   The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">\n"
"    SELinux Memory Protection Tests</a>\n"
"    web page explains this check.  This tool examined the library and it "
"looks\n"
"    like it was built correctly. So setroubleshoot can not determine if this\n"
"    application is compromised or not.  This could be a serious issue. Your\n"
"    system may very well be compromised.\n"
"\n"
"    Contact your security administrator and report this issue.\n"
"\n"
"    "
msgstr ""

#: ../src/allow_execmod.py:62
msgid ""
"\n"
"    If you trust $TARGET_PATH to run correctly, you can change the\n"
"    file context to textrel_shlib_t. \"chcon -t textrel_shlib_t\n"
"    '$TARGET_PATH'\"\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"textrel_shlib_t '$FIX_TARGET_PATH'\"\n"
"\n"
"    "
msgstr ""

#: ../src/allow_execstack.py:60
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from making the program stack "
"executable.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir que o $SOURCE_PATH torne executável a pilha do "
"programa.\n"
"    "

#: ../src/allow_execstack.py:64
msgid ""
"\n"
"    The $SOURCE application attempted to make its stack\n"
"    executable.  This is a potential security problem.  This should\n"
"    never ever be necessary. Stack memory is not executable on most\n"
"    OSes these days and this will not change. Executable stack memory\n"
"    is one of the biggest security problems. An execstack error might\n"
"    in fact be most likely raised by malicious code. Applications are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  If $SOURCE does not\n"
"    work and you need it to work, you can configure SELinux\n"
"    temporarily to allow this access until the application is fixed. Please\n"
"file a bug report.\n"
"    "
msgstr ""

#: ../src/allow_execstack.py:79
msgid ""
"\n"
"    Sometimes a library is accidentally marked with the execstack flag,\n"
"    if you find a library with this flag you can clear it with the\n"
"    execstack -c LIBRARY_PATH.  Then retry your application.  If the\n"
"    app continues to not work, you can turn the flag back on with\n"
"    execstack -s LIBRARY_PATH.\n"
"    "
msgstr ""

#: ../src/allow_execstack.py:89
msgid ""
"If you do not think $SOURCE_PATH should need to map stack memory that is "
"both writable and executable."
msgstr ""

#: ../src/allow_execstack.py:90
msgid "you need to report a bug. \n"
"This is a potentially dangerous access."
msgstr "precisa de reportar um erro.\n"
"Este acesso é potencialmente perigoso."

#: ../src/allow_execstack.py:99
#, python-format
msgid "If you believe that \n"
"%s\n"
"should not require execstack"
msgstr ""

#: ../src/allow_execstack.py:108
#, python-format
msgid ""
"you should clear the execstack flag and see if $SOURCE_PATH works correctly.\n"
"Report this as a bug on %s.\n"
"You can clear the exestack flag by executing:"
msgstr ""

#: ../src/allow_execstack.py:118
#, python-format
msgid "execstack -c %s"
msgstr "execstack -c %s"

#: ../src/allow_ftpd_use_cifs.py:29
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a CIFS "
"filesystem.\n"
"    "
msgstr ""

#: ../src/allow_ftpd_use_cifs.py:33
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a CIFS "
"filesystem.\n"
"    CIFS (Comment Internet File System) is a network filesystem similar to\n"
"    SMB (<a href=\"http://www.microsoft.com/mind/1196/cifs.asp\">http://www."
"microsoft.com/mind/1196/cifs.asp</a>)\n"
"    The ftp daemon attempted to read one or more files or directories from\n"
"    a mounted filesystem of this type.  As CIFS filesystems do not support\n"
"    fine-grained SELinux labeling, all files and directories in the\n"
"    filesystem will have the same security context.\n"
"\n"
"    If you have not configured the ftp daemon to read files from a CIFS "
"filesystem\n"
"    this access attempt could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/allow_ftpd_use_cifs.py:46
msgid ""
"\n"
"    Changing the \"$BOOLEAN\" boolean to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1.\"\n"
"    "
msgstr ""
"\n"
"    A alteração do booleano \"$BOOLEAN\" para 'true' irá permitir este "
"acesso:\n"
"    \"setsebool -P $BOOLEAN=1.\"\n"
"    "

#: ../src/allow_ftpd_use_cifs.py:53
msgid ""
" Changing the \"$BOOLEAN\" and\n"
"    \"$WRITE_BOOLEAN\" booleans to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1 $WRITE_BOOLEAN=1\".\n"
"    warning: setting the \"$WRITE_BOOLEAN\" boolean to true will\n"
"    allow the ftp daemon to write to all public content (files and\n"
"    directories with type public_content_t) in addition to writing to\n"
"    files and directories on CIFS filesystems.  "
msgstr ""
" A alteração dos booleanos \"$BOOLEAN\" e\n"
"    \"$WRITE_BOOLEAN\" para 'true' irá permitir este acesso:\n"
"    \"setsebool -P $BOOLEAN=1 $WRITE_BOOLEAN=1\".\n"
"    atenção: a definição do booleano \"$WRITE_BOOLEAN\" como 'true' irá\n"
"    permitir que o servidor de FTP escreva em todo o conteúdo público\n"
"    (pastas e ficheiros com o tipo 'public_content_t'), para além da\n"
"    gravação de pastas e ficheiros em sistemas de ficheiros CIFS.  "

#: ../src/allow_ftpd_use_cifs.py:62
msgid "If you want to allow ftpd to write to cifs file systems"
msgstr ""

#: ../src/allow_ftpd_use_cifs.py:63 ../src/allow_ftpd_use_nfs.py:65
msgid "you must tell SELinux about this"
msgstr "tem de informar o SELinux sobre isto"

#: ../src/allow_ftpd_use_nfs.py:29
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a NFS "
"filesystem.\n"
"    "
msgstr ""

#: ../src/allow_ftpd_use_nfs.py:33
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a NFS "
"filesystem.\n"
"    NFS (Network Filesystem) is a network filesystem commonly used on Unix / "
"Linux\n"
"    systems.\n"
"\n"
"    The ftp daemon attempted to read one or more files or directories from\n"
"    a mounted filesystem of this type.  As NFS filesystems do not support\n"
"    fine-grained SELinux labeling, all files and directories in the\n"
"    filesystem will have the same security context.\n"
"\n"
"    If you have not configured the ftp daemon to read files from a NFS "
"filesystem\n"
"    this access attempt could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/allow_ftpd_use_nfs.py:47
msgid ""
"\n"
"    Changing the \"allow_ftpd_use_nfs\" boolean to true will allow this "
"access:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1.\"\n"
"    "
msgstr ""
"\n"
"    A mudança do booleano \"allow_ftpd_use_nfs\" para 'true' irá permitir "
"este acesso:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1.\"\n"
"    "

#: ../src/allow_ftpd_use_nfs.py:54
msgid ""
" Changing the \"allow_ftpd_use_nfs\" and\n"
"    \"$WRITE_BOOLEAN\" booleans to true will allow this access:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1 $WRITE_BOOLEAN=1\".\n"
"    warning: setting the \"$WRITE_BOOLEAN\" boolean to true will\n"
"    allow the ftp daemon to write to all public content (files and\n"
"    directories with type public_content_t) in addition to writing to\n"
"    files and directories on NFS filesystems.  "
msgstr ""
" A mudança dos booleanos \"allow_ftpd_use_nfs\" e \"$WRITE_BOOLEAN\"\n"
"    para 'true' irá permitir este acesso:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1 $WRITE_BOOLEAN=1\".\n"
"    atenção: a definição do booleano \"$WRITE_BOOLEAN\" como 'true' irá\n"
"    permitir que o servidor de FTP escreva em todo o conteúdo público\n"
"    (pastas e ficheiros com o tipo 'public_content_t'), para além da\n"
"    gravação de pastas e ficheiros em sistemas de ficheiros NFS.  "

#: ../src/allow_ftpd_use_nfs.py:64
msgid "If you want to allow ftpd to write to nfs file systems"
msgstr ""

#: ../src/associate.py:32 ../src/openvpn.py:31 ../src/public_content.py:27
#: ../src/qemu_blk_image.py:27 ../src/qemu_file_image.py:27
#: ../src/restorecon.py:51 ../src/restorecon_source.py:33
#: ../src/rsync_data.py:27 ../src/sshd_root.py:32
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access to $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o $SOURCE_PATH \"$ACCESS\" acesso a "
"$TARGET_PATH.\n"
"    "

#: ../src/associate.py:37
#, python-format
msgid ""
"\n"
"You tried to place a type on a %s that is not a file type.  This is not "
"allowed, you must assigne a file type.  You can list all file types using "
"the seinfo command.\n"
"\n"
"seinfo -afile_type -x\n"
"\n"
"    "
msgstr ""

#: ../src/associate.py:44
#, python-format
msgid ""
"If you want to change the label of $TARGET_PATH to %s, you are not allowed "
"to since it is not a valid file type."
msgstr ""

#: ../src/associate.py:49
msgid "you must pick a valid file label."
msgstr ""

#: ../src/automount_exec_config.py:27
msgid ""
"\n"
"    SELinux is preventing the $SOURCE_PATH from executing potentially "
"mislabeled files $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o $SOURCE_PATH de executar ficheiros "
"$TARGET_PATH com etiquetas potencialmente inválidas.\n"
"    "

#: ../src/automount_exec_config.py:31
msgid ""
"\n"
"    SELinux has denied the $SOURCE_PATH from executing potentially\n"
"    mislabeled files $TARGET_PATH.  Automounter can be setup to execute\n"
"    configuration files. If $TARGET_PATH is an automount executable\n"
"    configuration file it needs to have a file label of bin_t.\n"
"    If automounter is trying to execute something that it is not supposed "
"to, this could indicate an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    O SELinux proibiu o $SOURCE_PATH de executar os ficheiros potencialmente\n"
"    mal etiquetados em $TARGET_PATH.  A montagem automática poderá ser "
"configurada\n"
"    para executar os ficheiros de configuração. Se o $TARGET_PATH é um "
"ficheiro\n"
"    de configuração executável, necessita de ter uma etiqueta de ficheiros "
"'bin_t'.\n"
"    Se a montagem automática estiver a tentar executar algo que não seja "
"suposto,\n"
"    isto poderá corresponder a uma tentativa de intrusão.\n"
"    "

#: ../src/automount_exec_config.py:39
msgid ""
"\n"
"    If you want to change the file context of $TARGET_PATH so that the "
"automounter can execute it you can execute \"chcon -t bin_t $TARGET_PATH\".  "
"If you want this to survive a relabel, you need to permanently change the "
"file context: execute  \"semanage fcontext -a -t bin_t '$FIX_TARGET_PATH'\".\n"
"    "
msgstr ""

#: ../src/bind_ports.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from binding to port $PORT_NUMBER.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o $SOURCE_PATH de se associar ao porto "
"$PORT_NUMBER.\n"
"    "

#: ../src/bind_ports.py:31
#, python-format
msgid ""
"\n"
"    SELinux has denied the $SOURCE from binding to a network port "
"$PORT_NUMBER which does not have an SELinux type associated with it.\n"
"    If $SOURCE should be allowed to listen on $PORT_NUMBER, use the "
"<i>semanage</i> command to assign $PORT_NUMBER to a port type that "
"$SOURCE_TYPE can bind to (%s).\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to bind to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/bind_ports.py:38
#, python-format
msgid ""
"\n"
"    If you want to allow $SOURCE to bind to port $PORT_NUMBER, you can "
"execute \n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    \n"
"where PORT_TYPE is one of the following: %s.\n"
"\n"
"    \n"
"\n"
"If this system is running as an NIS Client, turning on the allow_ypbind "
"boolean may fix the problem.  setsebool -P allow_ypbind=1.\n"
"    "
msgstr ""

#: ../src/bind_ports.py:47
msgid "If you want to allow $SOURCE_PATH to bind to network port $PORT_NUMBER"
msgstr ""

#: ../src/bind_ports.py:53 ../src/connect_ports.py:51
#, python-format
msgid ""
"# semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    where PORT_TYPE is one of the following: %s."
msgstr ""

#: ../src/bind_ports.py:56 ../src/connect_ports.py:54
#, python-format
msgid "# semanage port -a -t %s -p %s $PORT_NUMBER"
msgstr ""

#: ../src/catchall_boolean.py:32
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access on $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o $SOURCE_PATH \"$ACCESS\" de ter o acesso a "
"$TARGET_PATH.\n"
"    "

#: ../src/catchall_boolean.py:36
msgid ""
"\n"
"\n"
"    SELinux denied access requested by $SOURCE. The current boolean\n"
"    settings do not allow this access.  If you have not setup $SOURCE to\n"
"    require this access this may signal an intrusion attempt. If you do "
"intend\n"
"    this access you need to change the booleans on this system to allow\n"
"    the access.\n"
"    "
msgstr ""

#: ../src/catchall_boolean.py:45
msgid ""
"\n"
"    Confined processes can be configured to run requiring different access, "
"SELinux provides booleans to allow you to turn on/off\n"
"    access as needed.\n"
"\n"
"    "
msgstr ""

#: ../src/catchall_boolean.py:62
#, python-format
msgid "If you want to %s"
msgstr ""

#: ../src/catchall_boolean.py:65
#, python-format
msgid "setsebool -P %s %s"
msgstr "setsebool -P %s %s"

#: ../src/catchall_boolean.py:68
#, python-format
msgid "You must tell SELinux about this by enabling the '%s' boolean.\n"
msgstr ""

#: ../src/catchall_boolean.py:71
#, python-format
msgid "You can read '%s' man page for more details."
msgstr ""

#: ../src/catchall_labels.py:28
#, python-format
msgid ""
"\n"
"    SELinux has denied the $SOURCE access to potentially\n"
"    mislabeled files $TARGET_PATH.  This means that SELinux will not\n"
"    allow httpd to use these files. If httpd should be allowed this access "
"to these files you should change the file context to one of the following "
"types, %s.\n"
"    Many third party apps install html files\n"
"    in directories that SELinux policy cannot predict.  These directories\n"
"    have to be labeled with a file context which httpd can access.\n"
"    "
msgstr ""
"\n"
"    O SELinux negou o acesso de $SOURCE a ficheiros com legendas\n"
"    potencialmente inválidas em $TARGET_PATH.  Isto significa que o SELinux\n"
"    não irá permitir que o httpd use estes ficheiros. Se httpd deve ter "
"acesso\n"
"    a estes ficheiros você deve alterar o contexto do ficheiro para um dos\n"
"    seguintes tipos, %s. Muitas aplicações\n"
"    de terceiros instalam ficheiros html em pastas que a política do\n"
"    SELinux não consegue prever. Estas pastas deverão ser etiquetadas\n"
"    de novo com um contexto de ficheiro a que o httpd consiga aceder.\n"
"    "

#: ../src/catchall_labels.py:37
msgid "You need to change the label on $FIX_TARGET_PATH"
msgstr "Tem de mudar a etiqueta em $FIX_TARGET_PATH"

#: ../src/catchall_labels.py:40
#, python-format
msgid ""
"# semanage fcontext -a -t FILE_TYPE '$FIX_TARGET_PATH'\n"
"where FILE_TYPE is one of the following: %s.\n"
"Then execute:\n"
"restorecon -v '$FIX_TARGET_PATH'\n"
msgstr ""

#: ../src/catchall.py:29 ../src/chrome.py:29 ../src/mozplugger.py:29
#: ../src/mozplugger_remove.py:29
msgid "SELinux is preventing $SOURCE_PATH \"$ACCESS\" access."
msgstr ""

#: ../src/catchall.py:31
msgid ""
"\n"
"\n"
"    SELinux denied access requested by $SOURCE. It is not\n"
"    expected that this access is required by $SOURCE and this access\n"
"    may signal an intrusion attempt. It is also possible that the specific\n"
"    version or configuration of the application is causing it to require\n"
"    additional access.\n"
"\n"
"    "
msgstr ""
"\n"
"\n"
"    O SELinux negou o acesso pedido pelo $SOURCE. Não se espera que\n"
"    este acesso seja necessário para o $SOURCE e este acesso poderá\n"
"    assinalar uma tentativa de intrusão. Também é possível que a versão\n"
"    ou configuração específicas da aplicação estejam a obrigá-la a precisar\n"
"    de acesso adicional.\n"
"\n"
"    "

#: ../src/catchall.py:41
msgid ""
"\n"
"    You can generate a local policy module to allow this\n"
"    access - see <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>\n"
"\n"
"    Please file a bug report.\n"
"    "
msgstr ""
"\n"
"    Você pode gerar um módulo com uma politica local para permitir este\n"
"    acesso - Consulte <a href=\"http://docs.fedoraproject.org/selinux-faq-"
"fc5/#id2961385\">FAQ</a>\n"
"\n"
"    Por favor, preencha um relatório de erro.\n"
"    "

#: ../src/catchall.py:50
msgid ""
"If you believe that $SOURCE_BASE_PATH should be allowed $ACCESS access on "
"processes labeled $TARGET_TYPE by default."
msgstr ""

#: ../src/catchall.py:52
msgid ""
"If you believe that $SOURCE_BASE_PATH should have the $ACCESS capability by "
"default."
msgstr ""

#: ../src/catchall.py:53
msgid ""
"If you believe that $SOURCE_BASE_PATH should be allowed $ACCESS access on "
"the $TARGET_BASE_PATH $TARGET_CLASS by default."
msgstr ""

#: ../src/catchall.py:55
msgid ""
"You should report this as a bug.\n"
"You can generate a local policy module to allow this access."
msgstr ""

#: ../src/catchall.py:56
msgid ""
"Allow this access for now by executing:\n"
"# ausearch -c '$SOURCE' --raw | audit2allow -M my-$MODULE_NAME\n"
"# semodule -i my-$MODULE_NAME.pp"
msgstr ""

#: ../src/connect_ports.py:27 ../src/sandbox_connect.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from connecting to port $PORT_NUMBER.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o $SOURCE_PATH de se ligar ao porto "
"$PORT_NUMBER.\n"
"    "

#: ../src/connect_ports.py:31
#, python-format
msgid ""
"\n"
"    SELinux has denied $SOURCE from connecting to a network port "
"$PORT_NUMBER which does not have an SELinux type associated with it.\n"
"    If $SOURCE should be allowed to connect on $PORT_NUMBER, use the "
"<i>semanage</i> command to assign $PORT_NUMBER to a port type that "
"$SOURCE_TYPE can connect to (%s).\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to connect to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/connect_ports.py:38
#, python-format
msgid ""
"\n"
"    If you want to allow $SOURCE to connect to $PORT_NUMBER, you can execute "
"\n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"\n"
"    where PORT_TYPE is one of the following: %s.\n"
"    "
msgstr ""

#: ../src/connect_ports.py:45 ../src/sandbox_connect.py:47
msgid ""
"If you want to allow $SOURCE_PATH to connect to network port $PORT_NUMBER"
msgstr ""

#: ../src/chrome.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not expected that this "
"access is required by $SOURCE and this access may signal an intrusion "
"attempt. It is also possible that the specific version or configuration of "
"the application is causing it to require additional access.  spice-xpi run "
"applications within mozilla-plugins that require access to the desktop, that "
"the mozilla_plugin lockdown will not allow, so either you need to turn off "
"the mozilla_plugin lockdown or not use these packages.\n"
"    "
msgstr ""

#: ../src/chrome.py:35
msgid ""
"\n"
"Either remove the mozplugger or spice-xpi package by executing 'yum remove "
"mozplugger spice-xpi', or turn off enforcement of SELinux over the Chrome "
"plugins. setsebool -P unconfined_chrome_sandbox_transition 0\n"
"    "
msgstr ""

#: ../src/chrome.py:38 ../src/mozplugger.py:38
#, python-format
msgid "If you want to use the %s package"
msgstr ""

#: ../src/chrome.py:43
msgid "you must turn off SELinux controls on the Chrome plugins."
msgstr ""

#: ../src/cvs_data.py:27
msgid ""
"\n"
"    SELinux is preventing cvs ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o cvs ($SOURCE_PATH) \"$ACCESS\" de ter acesso "
"a $TARGET_PATH\n"
"    "

#: ../src/cvs_data.py:31
msgid ""
"\n"
"    SELinux denied cvs access to $TARGET_PATH.\n"
"    If this is a CVS repository it needs to have a file context label of\n"
"    cvs_data_t. If you did not intend to use $TARGET_PATH as a CVS "
"repository\n"
"    it could indicate either a bug or it could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    O SELinux negou o acesso do CVS ao $TARGET_PATH.\n"
"    Se este for um repositório de CVS, deverá ter uma etiqueta de contexto\n"
"    do ficheiro igual a 'cvs_data_t'. Se não pretender usar o $TARGET_PATH\n"
"    como repositório de CVS, poderá indicar tanto um erro como uma\n"
"    tentativa de intrusão.\n"
"    "

#: ../src/cvs_data.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t cvs_data_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"cvs_data_t '$FIX_TARGET_PATH'\"\n"
"    \n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando 'chcon -R -t cvs_data_t "
"'$TARGET_PATH'\n"
"    Também deve alterar os ficheiros predefinidos do contexto do ficheiro no "
"sistema para que sejam preservados em caso de uma re-etiquetagem completa.  "
"\"semanage fcontext -a -t vcs_data_t '$FIX_TARGET_PATH'\"\n"
"    \n"
"    "

#: ../src/cvs_data.py:44
msgid "If $TARGET_BASE_PATH should be shared via the cvs daemon"
msgstr ""

#: ../src/cvs_data.py:45 ../src/qemu_file_image.py:49
msgid "You need to change the label on $TARGET_BASE_PATH'"
msgstr "Tem de mudar a etiqueta em $TARGET_BASE_PATH"

#: ../src/dac_override.py:28
msgid "\n"
"    SELinux is preventing $SOURCE_PATH the \"$ACCESS\" capability.\n"
"    "
msgstr ""

#: ../src/dac_override.py:32
msgid ""
"\n"
"\tdac_override and dac_read_search capabilities usually indicates that the "
"root process does not have access to a file based on the permission flags.  "
"This usually mean you have some file with the wrong ownership/permissions on "
"it.\n"
"    "
msgstr ""

#: ../src/dac_override.py:38
msgid ""
"If you want to help identify if domain needs this access or you have a file "
"with the wrong permissions on your system"
msgstr ""

#: ../src/dac_override.py:39
msgid ""
"turn on full auditing to get path information about the offending file and "
"generate the error again."
msgstr ""

#: ../src/dac_override.py:40
msgid ""
"\n"
"Turn on full auditing\n"
"# auditctl -w /etc/shadow -p w\n"
"Try to recreate AVC. Then execute\n"
"# ausearch -m avc -ts recent\n"
"If you see PATH record check ownership/permissions on file, and fix it,\n"
"otherwise report as a bugzilla."
msgstr ""

#: ../src/device.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access to device "
"$TARGET_PATH.\n"
"    "
msgstr ""

#: ../src/device.py:33
msgid ""
"\n"
"\n"
"    SELinux has denied $SOURCE \"$ACCESS\" access to device $TARGET_PATH.\n"
"    $TARGET_PATH is mislabeled, this device has the default label of the /"
"dev directory, which should not\n"
"    happen.  All Character and/or Block Devices should have a label.\n"
"\n"
"    You can attempt to change the label of the file using\n"
"\n"
"    restorecon -v '$TARGET_PATH'.\n"
"\n"
"    If this device remains labeled device_t, then this is a bug in SELinux "
"policy.\n"
"\n"
"    Please file a bg report.\n"
"\n"
"    If you look at the other similar devices labels, ls -lZ /dev/SIMILAR, "
"and find a type that would work for $TARGET_PATH,\n"
"    you can use chcon -t SIMILAR_TYPE '$TARGET_PATH', If this fixes the "
"problem, you can make this permanent by executing\n"
"    semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"\n"
"    If the restorecon changes the context, this indicates that the "
"application that created the device, created it without\n"
"    using SELinux APIs.  If you can figure out which application created the "
"device, please file a bug report against this application.\n"
"\n"
"    "
msgstr ""

#: ../src/device.py:56
msgid ""
"\n"
"    Attempt restorecon -v '$TARGET_PATH' or chcon -t SIMILAR_TYPE "
"'$TARGET_PATH'\n"
"    "
msgstr ""
"\n"
"    Tente usar o comando 'restorecon -v $TARGET_PATH' ou 'chcon -t "
"TIPO_SEMELHANTE $TARGET_PATH'\n"
"    "

#: ../src/device.py:60
msgid ""
"You need to change the label on $TARGET_PATH to a type of a similar device."
msgstr ""

#: ../src/device.py:61
msgid ""
"# semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"
msgstr ""

#: ../src/disable_ipv6.py:30
msgid "\n"
"Disable IPV6 properly.\n"
"    "
msgstr "\n"
"Desactivar IPV6 convenientemente.\n"
"    "

#: ../src/disable_ipv6.py:40
msgid "If you want to disable IPV6 on this machine"
msgstr ""

#: ../src/disable_ipv6.py:41
msgid ""
"you need to set /proc/sys/net/ipv6/conf/all/disable_ipv6 to 1 and do not "
"blacklist the module'"
msgstr ""

#: ../src/disable_ipv6.py:42
msgid "Add \n"
"net.ipv6.conf.all.disable_ipv6 = 1\n"
"to /etc/sysctl.conf\n"
msgstr ""

#: ../src/file.py:27
msgid ""
"\n"
"    SELinux is preventing access to files with the label, file_t.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o acesso aos ficheiros com a etiqueta 'file_t'.\n"
"    "

#: ../src/file.py:31
msgid ""
"\n"
"    SELinux permission checks on files labeled file_t are being\n"
"    denied.  file_t is the context the SELinux kernel gives to files\n"
"    that do not have a label. This indicates a serious labeling\n"
"    problem. No files on an SELinux box should ever be labeled file_t.\n"
"    If you have just added a disk drive to the system you can\n"
"    relabel it using the restorecon command.  For example if you saved the\n"
"home directory from a previous installation that did not use SELinux, "
"'restorecon -R -v /home' will fix the labels.  Otherwise you should\n"
"    relabel the entire file system.\n"
"    "
msgstr ""

#: ../src/file.py:42
msgid ""
"\n"
"    You can execute the following command as root to relabel your\n"
"    computer system: \"touch /.autorelabel; reboot\"\n"
"    "
msgstr ""
"\n"
"    Poderá executar o seguinte comando como 'root' para voltar a etiquetar\n"
"    o seu sistema: \"touch /.autorelabel; reboot\"\n"
"    "

#: ../src/file.py:49
msgid "This is caused by a newly created file system."
msgstr ""

#: ../src/file.py:51
msgid "If you think this is caused by a badly mislabeled machine."
msgstr ""

#: ../src/file.py:55
msgid "you need to add labels to it."
msgstr "precisa de lhe adicionar etiquetas."

#: ../src/file.py:57
msgid "you need to fully relabel."
msgstr "precisa de etiquetar totalmente."

#: ../src/filesystem_associate.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from creating a file with a context "
"of $SOURCE_TYPE on a filesystem.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir que o $SOURCE_PATH crie um ficheiro com um "
"contexto $SOURCE_TYPE num sistema de ficheiros.\n"
"    "

#: ../src/filesystem_associate.py:31
msgid ""
"\n"
"    SELinux is preventing $SOURCE from creating a file with a context of "
"$SOURCE_TYPE on a filesystem.\n"
"    Usually this happens when you ask the cp command to maintain the context "
"of a file when\n"
"    copying between file systems, \"cp -a\" for example.  Not all file "
"contexts should be maintained\n"
"    between the file systems.  For example, a read-only file type like "
"iso9660_t should not be placed\n"
"    on a r/w system.  \"cp -p\" might be a better solution, as this will "
"adopt the default file context\n"
"    for the destination.\n"
"    "
msgstr ""

#: ../src/filesystem_associate.py:40
msgid ""
"\n"
"    Use a command like \"cp -p\" to preserve all permissions except SELinux "
"context.\n"
"    "
msgstr ""

#: ../src/filesystem_associate.py:43
msgid ""
"If you believe $SOURCE_BASE_PATH should be allowed to create "
"$TARGET_BASE_PATH files"
msgstr ""

#: ../src/filesystem_associate.py:44
msgid ""
"you need to use a different command. You are not allowed to preserve the "
"SELinux context on the target file system."
msgstr ""

#: ../src/filesystem_associate.py:46
msgid ""
"use a command like \"cp -p\" to preserve all permissions except SELinux "
"context."
msgstr ""
"utilize um comando como \"cp-p\" para preservar todas as permissões, excepto "
"o contexto SELinux."

#: ../src/httpd_can_sendmail.py:28
msgid "\n"
"    SELinux is preventing the http daemon from sending mail.\n"
"    "
msgstr "\n"
"    O SELinux está a impedir o serviço http de enviar correio.\n"
"    "

#: ../src/httpd_can_sendmail.py:32
msgid ""
"\n"
"    SELinux has denied the http daemon from sending mail. An\n"
"    httpd script is trying to connect to a mail port or execute the\n"
"    sendmail command. If you did not setup httpd to sendmail, this could\n"
"    signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/httpd_can_sendmail.py:39
msgid ""
"\n"
"    If you want httpd to send mail you need to turn on the\n"
"    $BOOLEAN boolean: \"setsebool -P\n"
"    $BOOLEAN=1\"\n"
"    "
msgstr ""
"\n"
"    Se quiser que o httpd envie correio precisa de activar\n"
"    o booleano $BOOLEAN: \"setsebool -P\n"
"    $BOOLEAN=1\"\n"
"    "

#: ../src/httpd_can_sendmail.py:47
msgid "If you want to allow httpd to send mail"
msgstr ""

#: ../src/httpd_can_sendmail.py:48
msgid "you must setup SELinux to allow this"
msgstr "tem de configurar o SELinux para permitir isto"

#: ../src/httpd_unified.py:29
msgid "\n"
"    SELinux prevented httpd $ACCESS access to http files.\n"
"    "
msgstr ""
"\n"
"    O SELinux impediu o 'httpd' de $ACCESS aos ficheiros de HTTP.\n"
"    "

#: ../src/httpd_unified.py:33
msgid ""
"\n"
"    SELinux prevented httpd $ACCESS access to http files.\n"
"\n"
"    Ordinarily httpd is allowed full access to all files labeled with http "
"file\n"
"    context.  This machine has a tightened security policy with the $BOOLEAN\n"
"    turned off,  This requires explicit labeling of all files.  If a file is\n"
"    a cgi script it needs to be labeled with httpd_TYPE_script_exec_t in "
"order\n"
"    to be executed.  If it is read only content, it needs to be labeled\n"
"    httpd_TYPE_content_t, it is writable content. it needs to be labeled\n"
"    httpd_TYPE_script_rw_t or httpd_TYPE_script_ra_t. You can use the\n"
"    chcon command to change these context.  Please refer to the man page\n"
"    \"man httpd_selinux\" or\n"
"    <a href=\"http://fedora.redhat.com/docs/selinux-apache-fc3\">FAQ</a>\n"
"    \"TYPE\" refers to one of \"sys\", \"user\" or \"staff\" or potentially "
"other\n"
"    script types.\n"
"    "
msgstr ""

#: ../src/httpd_unified.py:50
msgid ""
"\n"
"    Changing the \"$BOOLEAN\" boolean to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1\"\n"
"    "
msgstr ""
"\n"
"    A mudança do booleano \"$BOOLEAN\" para 'true' irá permitir este acesso:\n"
"    \"setsebool -P $BOOLEAN=1\"\n"
"    "

#: ../src/httpd_unified.py:57
msgid ""
"If you want to allow httpd to execute cgi scripts and to unify HTTPD "
"handling of all content files."
msgstr ""

#: ../src/httpd_unified.py:58
msgid ""
"you must tell SELinux about this by enabling the 'httpd_unified' and "
"'http_enable_cgi' booleans"
msgstr ""

#: ../src/httpd_write_content.py:30
msgid ""
"\n"
"    SELinux prevented httpd $ACCESS access to $TARGET_PATH.\n"
"\n"
"    httpd scripts are not allowed to write to content without explicit\n"
"    labeling of all files.  If $TARGET_PATH is writable content. it needs\n"
"    to be labeled httpd_sys_rw_content_t or if all you need is append you "
"can label it httpd_sys_ra_content_t.   Please refer to 'man httpd_selinux' "
"for more information on setting up httpd and selinux.\n"
"    "
msgstr ""

#: ../src/httpd_write_content.py:40 ../src/qemu_blk_image.py:45
#: ../src/samba_share.py:45 ../src/swapfile.py:45 ../src/xen_image.py:48
msgid "You need to change the label on '$FIX_TARGET_PATH'"
msgstr "tem de alterar a etiqueta em '$FIX_TARGET_PATH'"

#: ../src/kernel_modules.py:29
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to modify "
"kernel configuration.\n"
"    "
msgstr ""
"\n"
"    O seu sistema pode estar seriamente comprometido! $SOURCE_PATH tentou "
"modificar a configuração do kernel.\n"
"    "

#: ../src/kernel_modules.py:33
msgid ""
"\n"
"    SELinux has prevented $SOURCE from modifying $TARGET.  This denial\n"
"    indicates $SOURCE was trying to modify the way the kernel runs or to\n"
"    actually insert code into the kernel. All applications that need this\n"
"    access should have already had policy written for them.  If a "
"compromised\n"
"    application tries to modify the kernel this AVC will be generated. This "
"is a\n"
"    serious issue. Your system may very well be compromised.\n"
"    "
msgstr ""

#: ../src/kernel_modules.py:44 ../src/selinuxpolicy.py:44
msgid ""
"If you do not think $SOURCE_BASE_PATH should try $ACCESS access on "
"$TARGET_BASE_PATH."
msgstr ""

#: ../src/kernel_modules.py:45 ../src/selinuxpolicy.py:45
msgid ""
"you may be under attack by a hacker, since confined applications should not "
"need this access."
msgstr ""

#: ../src/leaks.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH access to a leaked $TARGET_PATH file "
"descriptor.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir que o $SOURCE_PATH aceda a um descritor "
"$TARGET_PATH de ficheiro com uma fuga.\n"
"    "

#: ../src/leaks.py:31
msgid ""
"\n"
"    SELinux denied access requested by the $SOURCE command. It looks like "
"this is either a leaked descriptor or $SOURCE output was redirected to a "
"file it is not allowed to access.  Leaks usually can be ignored since "
"SELinux is just closing the leak and reporting the error.  The application "
"does not use the descriptor, so it will run properly.  If this is a "
"redirection, you will not get output in the $TARGET_PATH.  You should "
"generate a bugzilla on selinux-policy, and it will get routed to the "
"appropriate package.  You can safely ignore this avc.\n"
"    "
msgstr ""
"\n"
"    O SELinux negou o acesso pedido pelo comando $SOURCE. Isto parece que "
"houve uma fuga num apontador ou a saída de $SOURCE foi redireccionada para "
"um ficheiro que não está autorizado a aceder.  As fugas podem ser ignoradas, "
"uma vez que o SELinux está apenas a fechar a fuga e a reportar o erro.  A "
"aplicação não utiliza o apontador, por isso, irá executar normalmente.  Se "
"isto é um redireccionamento, você não receberá os dados da saída em "
"$TARGET_PATH.  Você deveria gerar um relatório de erro no bugzilla sobre o "
"selinux-policy. Ele será encaminhado para o pacote correcto.  Você pode, "
"tranquilamente, ignorar este avc.\n"
"    "

#: ../src/leaks.py:35
msgid ""
"\n"
"    You can generate a local policy module to allow this\n"
"    access - see <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>\n"
"    "
msgstr ""
"\n"
"    Você pode gerar um módulo com uma politica local para permitir este\n"
"    acesso - Consulte <a href=\"http://docs.fedoraproject.org/selinux-faq-"
"fc5/#id2961385\">FAQ</a>\n"
"    "

#: ../src/leaks.py:42
msgid ""
"If you want to ignore $SOURCE_BASE_PATH trying to $ACCESS access the "
"$TARGET_BASE_PATH $TARGET_CLASS, because you believe it should not need this "
"access."
msgstr ""

#: ../src/leaks.py:43
msgid ""
"You should report this as a bug.  \n"
"You can generate a local policy module to dontaudit this access."
msgstr ""

#: ../src/leaks.py:44
msgid ""
"# ausearch -x $SOURCE_PATH --raw | audit2allow -D -M my-$SOURCE\n"
"# semodule -i my-$SOURCE.pp"
msgstr ""

#: ../src/mmap_zero.py:29
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH attempted to mmap "
"low kernel memory.\n"
"    "
msgstr ""
"\n"
"    O seu sistema pode estar seriamente comprometido! $SOURCE_PATH tentou "
"endereçar (nmap) baixa memória do kernel.\n"
"    "

#: ../src/mmap_zero.py:33
msgid ""
"\n"
"    SELinux has denied the $SOURCE the ability to mmap low area of the "
"kernel \n"
"    address space.  The ability to mmap a low area of the address space is \n"
"    configured by /proc/sys/kernel/mmap_min_addr.  Preventing such mappings \n"
"    helps protect against exploiting null deref bugs in the kernel. All \n"
"    applications that need this access should have already had policy "
"written \n"
"    for them.  If a compromised application tries to modify the kernel, this "
"AVC \n"
"    would be generated. This is a serious issue. Your system may very well "
"be \n"
"    compromised.\n"
"    "
msgstr ""

#: ../src/mmap_zero.py:46
msgid ""
"If you do not think $SOURCE_PATH should need to mmap low memory in the "
"kernel."
msgstr ""

#: ../src/mmap_zero.py:47
msgid "you may be under attack by a hacker, this is a very dangerous access."
msgstr "pode estar sobre ataque de um hacker. Este acesso é muito perigoso."

#: ../src/mounton.py:29
msgid ""
"\n"
"    SELinux prevented $SOURCE from mounting on the file or directory\n"
"    \"$TARGET_PATH\" (type \"$TARGET_TYPE\").\n"
"    "
msgstr ""
"\n"
"    O SELinux impediu o $SOURCE de montar na pasta ou ficheiro\n"
"    \"$TARGET_PATH\" (tipo \"$TARGET_TYPE\").\n"
"    "

#: ../src/mounton.py:34
msgid ""
"\n"
"    SELinux prevented $SOURCE from mounting a filesystem on the file\n"
"    or directory \"$TARGET_PATH\" of type \"$TARGET_TYPE\". By default\n"
"    SELinux limits the mounting of filesystems to only some files or\n"
"    directories (those with types that have the mountpoint attribute). The\n"
"    type \"$TARGET_TYPE\" does not have this attribute. You can change the\n"
"    label of the file or directory.\n"
"    "
msgstr ""

#: ../src/mounton.py:43
msgid ""
"\n"
"    Changing the file_context to mnt_t will allow mount to mount the file "
"system:\n"
"    \"chcon -t mnt_t '$TARGET_PATH'.\"\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"mnt_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Mudando o contexto de file_context para mnt_t irá permitir ao comando "
"mount\n"
"    montar o sistema de ficheiros:\n"
"    \"chcon -t mnt_t '$TARGET_PATH'.\"\n"
"   Também precisa de alterar o contexto por omissão dos ficheiros do sistema "
"para os preservar mesmo no caso de etiquetagem total.  \"semanage fcontext -"
"a -t mnt_t '$FIX_TARGET_PATH'\"\n"
"   "

#: ../src/mounton.py:48
msgid "If you want to allow $SOURCE_BASE_PATH to mount on $TARGET_BASE_PATH."
msgstr ""

#: ../src/mounton.py:49
msgid "you must change the labeling on $TARGET_PATH."
msgstr "tem de alterar a etiquetagem em $TARGET_PATH."

#: ../src/mozplugger.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not expected that this "
"access is required by $SOURCE and this access may signal an intrusion "
"attempt. It is also possible that the specific version or configuration of "
"the application is causing it to require additional access.  mozplugger and "
"spice-xpi run applications within mozilla-plugins that require access to the "
"desktop, that the mozilla_plugin lockdown will not allow, so either you need "
"to turn off the mozilla_plugin lockdown or not use these packages.\n"
"    "
msgstr ""

#: ../src/mozplugger.py:35
msgid ""
"\n"
"Either remove the mozplugger or spice-xpi package by executing 'yum remove "
"mozplugger spice-xpi' or turn off enforcement of SELinux over the Firefox "
"plugins. setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "
msgstr ""

#: ../src/mozplugger.py:43
msgid "you must turn off SELinux controls on the Firefox plugins."
msgstr ""

#: ../src/mozplugger_remove.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not\n"
"    expected that this access is required by $SOURCE and this access\n"
"    may signal an intrusion attempt. It is also possible that the specific\n"
"    version or configuration of the application is causing it to require\n"
"    additional access.\n"
"    "
msgstr ""

#: ../src/mozplugger_remove.py:39
msgid ""
"\n"
"Either remove the mozplluger package by executing 'yum remove mozplugger'\n"
"Or turn off enforcement of SELinux over the Firefox plugins.\n"
"setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "
msgstr ""

#: ../src/mozplugger_remove.py:44
msgid ""
"If you want to to continue using SELinux Firefox plugin containment rather "
"then using mozplugger package"
msgstr ""

#: ../src/mozplugger_remove.py:45
msgid "you must remove the mozplugger package."
msgstr ""

#: ../src/openvpn.py:35
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled. openvpn is allowed to read content in home directory if "
"it\n"
"    is labeled correctly.\n"
"    "
msgstr ""

#: ../src/openvpn.py:41
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  \n"
"    # restorecon -R /root/.ssh\n"
"    "
msgstr ""

#: ../src/openvpn.py:49
msgid ""
"If you want to mv $TARGET_BASE_PATH to standard location so that "
"$SOURCE_BASE_PATH can have $ACCESS access"
msgstr ""

#: ../src/openvpn.py:51
msgid ""
"If you want to modify the label on $TARGET_BASE_PATH so that "
"$SOURCE_BASE_PATH can have $ACCESS access on it"
msgstr ""

#: ../src/openvpn.py:55
msgid "you must move the cert file to the ~/.cert directory"
msgstr ""

#: ../src/openvpn.py:57 ../src/sshd_root.py:48
msgid "you must fix the labels."
msgstr "precisa de corrigir as etiquetas."

#: ../src/public_content.py:31
msgid ""
"\n"
"    SELinux denied access to $TARGET_PATH requested by $SOURCE.\n"
"    $TARGET_PATH has a context used for sharing by a different program. If "
"you\n"
"    would like to share $TARGET_PATH from $SOURCE also, you need to\n"
"    change its file context to public_content_t.  If you did not intend to\n"
"    allow this access, this could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    O SELinux proibiu o acesso ao $TARGET_PATH, pedido pelo $SOURCE.\n"
"    O $TARGET_PATH tem um contexto usado para partilhar por vários programas."
"\n"
"    Se quiser partilhar o $TARGET_PATH também a partir de $SOURCE,\n"
"    terá de mudar o seu contexto de ficheiro para 'public_content_t'.\n"
"    Se não pretende este acesso, isto poderá assinalar uma tentativa de\n"
"    intrusão.\n"
"    "

#: ../src/public_content.py:39
msgid ""
"\n"
"    You can alter the file context by executing chcon -t public_content_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"public_content_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando chcon -t public_content_t "
"'$TARGET_PATH'\n"
"    Também precisa de alterar o contexto por omissão dos ficheiros no "
"sistema para os preservar mesmo na caso de uma etiquetagem total.  "
"\"semanage fcontext -a -t public_content_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/public_content.py:50
msgid "If you want to treat $TARGET_BASE_PATH as public content"
msgstr ""

#: ../src/public_content.py:51
msgid ""
"You need to change the label on $TARGET_BASE_PATH to public_content_t or "
"public_content_rw_t."
msgstr ""

#: ../src/qemu_blk_image.py:31
msgid ""
"\n"
"    SELinux denied qemu access to the block device $TARGET_PATH.\n"
"    If this is a virtualization image, it needs to be labeled with a "
"virtualization file context (virt_image_t). You can relabel $TARGET_PATH to "
"be virt_image_t using chcon.  You also need to execute semanage fcontext -a -"
"t virt_image_t '$FIX_TARGET_PATH' to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a qemu\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    O SELinux negou o acesso do qemu ao dispositivo de bloco $TARGET_PATH.\n"
"    Se isto é uma imagem virtual, então precisa de ter uma etiqueta de "
"contexto de ficheiro virtual (virt_image_t). Poderá etiquetar de novo o "
"$TARGET_PATH para ser virt_image_t utilizando chcon. Se o fizer, deverá "
"também executar o comando 'semanage fcontext -a -t virt_image_t "
"$FIX_TARGET_PATH' para adicionar este\n"
"    novo caminho às predefinições do sistema. Se não pretendia usar o "
"$TARGET_PATH como uma imagem\n"
"   qemu, tanto poderá indicar um erro ou uma tentativa de intrusão.\n"
"   "

#: ../src/qemu_blk_image.py:38 ../src/qemu_file_image.py:41
msgid ""
"\n"
"    You can alter the file context by executing chcon -t virt_image_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"virt_image_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando chcon -t virt_image_t "
"'$TARGET_PATH'\n"
"    Também precisa de alterar o contexto por omissão dos ficheiros no "
"sistema para os preservar mesmo na caso de uma etiquetagem total.  "
"\"semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/qemu_blk_image.py:46
msgid ""
"# semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"
msgstr ""

#: ../src/qemu_file_image.py:31
msgid ""
"\n"
"    SELinux denied qemu access to $TARGET_PATH.\n"
"    If this is a virtualization image, it has to have a file context label "
"of\n"
"    virt_image_t. The system is setup to label image files in directory./var/"
"lib/libvirt/images\n"
"    correctly.  We recommend that you copy your image file to /var/lib/"
"libvirt/images.\n"
"    If you really want to have your qemu image files in the current "
"directory, you can relabel $TARGET_PATH to be virt_image_t using chcon.  You "
"also need to execute semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH' "
"to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a qemu\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    O SELinux negou o acesso do qemu a $TARGET_PATH.\n"
"    Se esta for uma imagem virtual deverá ter uma etiqueta do contexto de\n"
"    ficheiros 'virt_image_t'. O sistema está configurado para etiquetar os\n"
"    ficheiros das imagens correctamente em /var/lib/libvirt/images. "
"Recomenda-se\n"
"    que copie o ficheiro da sua imagem para esta pasta.\n"
"    Se quiser ter de facto as suas imagens qemu na pasta actual, poderá "
"etiquetar\n"
"    de novo o $TARGET_PATH como sendo 'virt_image_t', usando\n"
"    o comando 'chcon'. Se o fizer, deverá também executar o comando\n"
"    'semanage fcontext -a -t virt_image_t $FIX_TARGET_PATH' para adicionar "
"este\n"
"    local novo às predefinições do sistema. Se não pretendia usar\n"
"    o $TARGET_PATH como uma imagem do qemu, tanto poderá indicar um erro\n"
"    ou uma tentativa de intrusão.\n"
"    "

#: ../src/qemu_file_image.py:48
msgid "If $TARGET_BASE_PATH is a virtualization target"
msgstr ""

#: ../src/restorecon.py:57
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  restorecon '$TARGET_PATH', if this file is a "
"directory,\n"
"    you can recursively restore using restorecon -R '$TARGET_PATH'.\n"
"    "
msgstr ""
"\n"
"    Pode restaurar o contexto do sistema padrão para este ficheiro "
"executandoo\n"
"    comando restorecon.  restorecon '$TARGET_PATH', se este ficheiro for um "
"directório,\n"
"    pode restaurar recursivamente usando restorecon -R '$TARGET_PATH'.\n"
"    "

#: ../src/restorecon.py:64
#, python-format
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled.  $TARGET_PATH default SELinux type is\n"
"    <B>%s</B>, but its current type is <B>$TARGET_TYPE</B>. Changing\n"
"    this file back to the default type may fix your problem.\n"
"    <p>\n"
"    File contexts can be assigned to a file in the following ways.\n"
"    <ul>\n"
"        <li>Files created in a directory receive the file context of the "
"parent directory by default.\n"
"        <li>The SELinux policy might override the default label inherited "
"from the parent directory by\n"
"            specifying a process running in context A which creates a file "
"in a directory labeled B\n"
"            will instead create the file with label C. An example of this "
"would be the dhcp client running\n"
"            with the dhclient_t type and creating a file in the directory /"
"etc. This file would normally\n"
"            receive the etc_t type due to parental inheritance but instead "
"the file\n"
"            is labeled with the net_conf_t type because the SELinux policy "
"specifies this.\n"
"        <li>Users can change the file context on a file using tools such as "
"chcon, or restorecon.\n"
"    </ul>\n"
"    This file could have been mislabeled either by user error, or if an "
"normally confined application\n"
"    was run under the wrong domain.\n"
"    <p>\n"
"    However, this might also indicate a bug in SELinux because the file "
"should not have been labeled\n"
"    with this type.\n"
"    <p>\n"
"    If you believe this is a bug, please file a bug report against this "
"package.\n"
"    "
msgstr ""

#: ../src/restorecon.py:90
#, python-format
msgid ""
"If you want to fix the label. \n"
"$TARGET_PATH default label should be %s."
msgstr ""

#: ../src/restorecon.py:96
msgid ""
"you can run restorecon. The access attempt may have been stopped due to "
"insufficient permissions to access a parent directory in which case try to "
"change the following command accordingly."
msgstr ""

#: ../src/restorecon.py:105 ../src/restorecon_source.py:73
msgid "Restore\n"
"Context"
msgstr ""

#: ../src/restorecon_source.py:39
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  restorecon '$SOURCE_PATH'.\n"
"    "
msgstr ""

#: ../src/restorecon_source.py:45
#, python-format
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $SOURCE_PATH may\n"
"    be mislabeled.  $SOURCE_PATH default SELinux type is\n"
"    <B>%s</B>, but its current type is <B>$SOURCE_TYPE</B>. Changing\n"
"    this file back to the default type may fix your problem.\n"
"    <p>\n"
"    This file could have been mislabeled either by user error, or if an "
"normally confined application\n"
"    was run under the wrong domain.\n"
"    <p>\n"
"    However, this might also indicate a bug in SELinux because the file "
"should not have been labeled\n"
"    with this type.\n"
"    <p>\n"
"    If you believe this is a bug, please file a bug report against this "
"package.\n"
"    "
msgstr ""

#: ../src/restorecon_source.py:60
#, python-format
msgid ""
"If you want to fix the label. \n"
"$SOURCE_PATH default label should be %s."
msgstr ""

#: ../src/restorecon_source.py:65
msgid "you can run restorecon."
msgstr ""

#: ../src/rsync_data.py:31
msgid ""
"\n"
"    SELinux denied RSYNC access to $TARGET_PATH.\n"
"    If this is an RSYNC repository, it has to have a file context label of\n"
"    rsync_data_t. If you did not intend to use $TARGET_PATH as an RSYNC "
"repository,\n"
"    this message could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/rsync_data.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t rsync_data_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"rsync_data_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando chcon -R -t rsync_data_t "
"'$TARGET_PATH'\n"
"    Também precisa de alterar o contexto por omissão dos ficheiros no "
"sistema para os preservar mesmo na caso de uma etiquetagem total.  "
"\"semanage fcontext -a -t rsync_data_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/rsync_data.py:45
msgid "If $TARGET_BASE_PATH should be shared via the RSYNC daemon"
msgstr ""

#: ../src/rsync_data.py:46
msgid "You need to change the label on $TARGET_BASE_PATH"
msgstr ""

#: ../src/samba_share.py:27
msgid ""
"\n"
"    SELinux is preventing Samba ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o Samba ($SOURCE_PATH) \"$ACCESS\" de ter o "
"acesso a $TARGET_PATH.\n"
"    "

#: ../src/samba_share.py:31
msgid ""
"\n"
"    SELinux denied samba access to $TARGET_PATH.\n"
"    If you want to share this directory with samba it has to have a file "
"context label of\n"
"    samba_share_t. If you did not intend to use $TARGET_PATH as a samba "
"repository,\n"
"    this message could indicate either a bug or an intrusion attempt.\n"
"    Please refer to 'man samba_selinux' for more information on setting up "
"Samba and SELinux.\n"
"    "
msgstr ""

#: ../src/samba_share.py:39
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t samba_share_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"samba_share_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando chcon -R -t samba_share_t "
"'$TARGET_PATH'\n"
"    Também precisa de alterar o contexto por omissão dos ficheiros no "
"sistema para os preservar mesmo na caso de uma etiquetagem total.  "
"\"semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/samba_share.py:53
#, python-format
msgid ""
"# semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH%s'\n"
"# restorecon %s -v '$FIX_TARGET_PATH'"
msgstr ""

#: ../src/sandbox_connect.py:33
msgid ""
"\n"
"    SELinux has denied $SOURCE from connecting to a network port "
"$PORT_NUMBER within a sandbox.\n"
"    If $SOURCE should be allowed to connect on $PORT_NUMBER, you need to use "
"a different sandbox type like sandbox_web_t or sandbox_net_t.  \n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to connect to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/sandbox_connect.py:41
msgid ""
"\n"
"    If you want to allow $SOURCE to connect to $PORT_NUMBER, you can execute "
"\n"
"\n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    "
msgstr ""

#: ../src/sandbox_connect.py:49
msgid ""
"you need to modify the sandbox type. sandbox_web_t or sandbox_net_t.\n"
"For example:\n"
"sandbox -X -t sandbox_net_t $SOURCE_PATH\n"
"Please read 'sandbox' man page for more details.\n"
msgstr ""

#: ../src/selinuxpolicy.py:29
msgid "\n"
"    Your system may be seriously compromised!\n"
"    "
msgstr "\n"
"    O seu sistema pode estar seriamente comprometido!\n"
"    "

#: ../src/selinuxpolicy.py:33
msgid ""
"\n"
"    SELinux has prevented $SOURCE from modifying $TARGET.  This denial\n"
"    indicates $SOURCE was trying to modify the selinux policy configuration.\n"
"    All applications that need this access should have already had policy\n"
"    written for them.  If a compromised application tries to modify the "
"SELinux\n"
"    policy this AVC will be generated. This is a serious issue. Your system\n"
"    may very well be compromised.\n"
"    "
msgstr ""

#: ../src/setenforce.py:27
msgid "If you did not directly cause this AVC through testing."
msgstr ""

#: ../src/setenforce.py:28
msgid "if you think that you might have been hacked"
msgstr ""

#: ../src/setenforce.py:32
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to modify "
"SELinux enforcement.\n"
"    "
msgstr ""
"\n"
"    O seu sistema pode estar seriamente comprometido! $SOURCE_PATH tentou "
"modificar a politica do SELinux.\n"
"    "

#: ../src/setenforce.py:36
msgid ""
"\n"
"    SELinux has prevented $SOURCE from writing to a file under /sys/fs/"
"selinux.\n"
"    Files under /sys/fs/selinux control the way SELinux is configured.\n"
"    All programs that need to write to files under /sys/fs/selinux should "
"have already had policy\n"
"    written for them. If a compromised application tries to turn off SELinux\n"
"    this AVC will be generated. This is a serious issue. Your system may "
"very\n"
"    well be compromised.\n"
"    "
msgstr ""

#: ../src/setenforce.py:48
msgid "If you believe $SOURCE_PATH tried to disable SELinux."
msgstr ""

#: ../src/setenforce.py:49
msgid ""
"you may be under attack by a hacker, since confined applications should "
"never need this access."
msgstr ""

#: ../src/sshd_root.py:36
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled. sshd is allowed to read content in /root/.ssh directory "
"if it\n"
"    is labeled correctly.\n"
"    "
msgstr ""

#: ../src/sshd_root.py:42
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.\n"
"    # restorecon -R /root/.ssh\n"
"    "
msgstr ""

#: ../src/sshd_root.py:55
msgid "Restore Context"
msgstr "Restaurar Contexto"

#: ../src/swapfile.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" to $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o $SOURCE_PATH \"$ACCESS\" ao $TARGET_PATH.\n"
"    "

#: ../src/swapfile.py:31
msgid ""
"\n"
"    SELinux denied $SOURCE access to $TARGET_PATH.\n"
"    If this is a swapfile, it has to have a file context label of\n"
"    swapfile_t. If you did not intend to use\n"
"    $TARGET_PATH as a swapfile, this message could indicate either a bug or "
"an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/swapfile.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -t swapfile_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"swapfile_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando chcon -t swapfile_t "
"'$TARGET_PATH'\n"
"    Também precisa de alterar o contexto por omissão dos ficheiros no "
"sistema para os preservar mesmo na caso de uma etiquetagem total.  "
"\"semanage fcontext -a -t swapfile_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/sys_module.py:28
msgid ""
"If you do not believe your $SOURCE_PATH should be modifying the kernel, by "
"loading kernel modules"
msgstr ""

#: ../src/sys_module.py:29
msgid "You might have been hacked."
msgstr ""

#: ../src/sys_module.py:33
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to load a "
"kernel module.\n"
"    "
msgstr ""
"\n"
"    O seu sistema pode estar seriamente comprometido! $SOURCE_PATH tentou "
"carregar um módulo do kernel.\n"
"    "

#: ../src/sys_module.py:37
msgid ""
"\n"
"    SELinux has prevented $SOURCE from loading a kernel module.\n"
"    All confined programs that need to load kernel modules should have "
"already had policy\n"
"    written for them. If a compromised application\n"
"    tries to modify the kernel this AVC will be generated. This is a serious\n"
"    issue. Your system may very well be compromised.\n"
"    "
msgstr ""

#: ../src/sys_module.py:46
msgid ""
"If you do not believe that $SOURCE_PATH should be attempting to modify the "
"kernel by loading a kernel module."
msgstr ""

#: ../src/sys_module.py:47
msgid "A process might be attempting to hack into your system."
msgstr ""

#: ../src/sys_resource.py:28
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH the \"sys_resource\" capability.\n"
"    "
msgstr ""

#: ../src/sys_resource.py:32
msgid ""
"\n"
"    Confined domains should not require \"sys_resource\". This usually means "
"that     your system is running out some system resource like disk space, "
"memory, quota etc. Please clear up the disk and this\n"
"    AVC message should go away. If this AVC continues after you clear up the "
"disk space, please report this as a bug.\n"
"    "
msgstr ""

#: ../src/sys_resource.py:39
msgid ""
"If you do not want processes to require capabilities to use up all the "
"system resources on your system;"
msgstr ""

#: ../src/sys_resource.py:40
msgid ""
"you need to diagnose why your system is running out of system resources and "
"fix the problem.\n"
"\n"
"According to /usr/include/linux/capability.h, sys_resource is required to:\n"
"\n"
"/* Override resource limits. Set resource limits. */\n"
"/* Override quota limits. */\n"
"/* Override reserved space on ext2 filesystem */\n"
"/* Modify data journaling mode on ext3 filesystem (uses journaling\n"
"   resources) */\n"
"/* NOTE: ext2 honors fsuid when checking for resource overrides, so\n"
"   you can override using fsuid too */\n"
"/* Override size restrictions on IPC message queues */\n"
"/* Allow more than 64hz interrupts from the real-time clock */\n"
"/* Override max number of consoles on console allocation */\n"
"/* Override max number of keymaps */\n"
msgstr ""

#: ../src/vbetool.py:30
msgid ""
"\n"
"    SELinux has prevented vbetool from performing an unsafe memory operation."
"\n"
"    "
msgstr ""
"\n"
"    O SELinux impediu o vbetool de efectuar uma operação de memória insegura."
"\n"
"    "

#: ../src/vbetool.py:34
msgid ""
"\n"
"SELinux denied an operation requested by $SOURCE, a program used\n"
"to alter video hardware state.  This program is known to use\n"
"an unsafe operation on system memory but so are a number of\n"
"malware/exploit programs which masquerade as vbetool.  This tool is used to\n"
"reset video state when a machine resumes from a suspend.  If your machine\n"
"is not resuming properly your only choice is to allow this\n"
"operation and reduce your system security against such malware.\n"
"\n"
"    "
msgstr ""

#: ../src/vbetool.py:45 ../src/wine.py:56
msgid ""
"\n"
"If you decide to continue to run the program in question you will need\n"
"to allow this operation.  This can be done on the command line by\n"
"executing:\n"
"\n"
"# setsebool -P mmap_low_allowed 1\n"
msgstr ""
"\n"
"Se decidir continuar a executar este programa, então terá de\n"
"permitir esta operação.  Isto pode ser feito na linha de comando\n"
"executando:\n"
"\n"
"# setsebool -P mmap_low_allowed 1\n"

#: ../src/vbetool.py:55
msgid ""
"If you want to ignore this AVC because it is dangerous and your machine "
"seems to be working correctly."
msgstr ""

#: ../src/vbetool.py:56
msgid ""
"you must tell SELinux about this by enabling the vbetool_mmap_zero_ignore "
"boolean."
msgstr ""

#: ../src/vbetool.py:62 ../src/wine.py:73
msgid "Turn off memory protection"
msgstr "Desligar protecção de memória"

#: ../src/wine.py:31
msgid ""
"\n"
"    SELinux has prevented wine from performing an unsafe memory operation.\n"
"    "
msgstr ""
"\n"
"    O SELinux impediu o wine de efectuar uma operação de memória insegura.\n"
"    "

#: ../src/wine.py:35
msgid ""
"\n"
"SELinux denied an operation requested by wine-preloader, a program used\n"
"to run Windows applications under Linux.  This program is known to use\n"
"an unsafe operation on system memory but so are a number of\n"
"malware/exploit programs which masquerade as wine.  If you were\n"
"attempting to run a Windows program your only choices are to allow this\n"
"operation and reduce your system security against such malware or to\n"
"refrain from running Windows applications under Linux.  If you were not\n"
"attempting to run a Windows application this indicates you are likely\n"
"being attacked by some for of malware or program trying to exploit your\n"
"system for nefarious purposes.\n"
"\n"
"Please refer to\n"
"\n"
"http://wiki.winehq.org/PreloaderPageZeroProblem\n"
"\n"
"Which outlines the other problems wine encounters due to its unsafe use\n"
"of memory and solutions to those problems.\n"
"\n"
"    "
msgstr ""

#: ../src/wine.py:66
msgid ""
"If you want to ignore this AVC because it is dangerous and your wine "
"applications are working correctly."
msgstr ""

#: ../src/wine.py:67
msgid ""
"you must tell SELinux about this by enabling the wine_mmap_zero_ignore "
"boolean."
msgstr ""

#: ../src/xen_image.py:27
msgid ""
"\n"
"    SELinux is preventing xen ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    O SELinux está a impedir o xen ($SOURCE_PATH) \"$ACCESS\" de ter o "
"acesso a $TARGET_PATH.\n"
"    "

#: ../src/xen_image.py:31
msgid ""
"\n"
"    SELinux denied xen access to $TARGET_PATH.\n"
"    If this is a XEN image, it has to have a file context label of\n"
"    xen_image_t. The system is setup to label image files in directory /var/"
"lib/xen/images\n"
"    correctly.  We recommend that you copy your image file to /var/lib/xen/"
"images.\n"
"    If you really want to have your xen image files in the current "
"directory, you can relabel $TARGET_PATH to be xen_image_t using chcon.  You "
"also need to execute semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH' "
"to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a xen\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    O SELinux negou o acesso do xen a $TARGET_PATH.\n"
"    Se esta for uma imagem XEN deverá ter uma etiqueta de contexto de \n"
"    ficheiros 'xen_image_t'. O sistema está configurado para etiquetar os\n"
"    ficheiros das imagens correctamente em /var/lib/xen/image. Recomenda-se\n"
"    que copie o ficheiro da sua imagem para esta pasta.  Se quiser ter de\n"
"    facto os seus ficheiros de imagens xen na pasta actual, poderá etiquetar\n"
"    de novo o $TARGET_PATH como sendo um ficheiro/pasta 'xen_image_t', "
"usando\n"
"    o comando 'chcon'. Se o fizer, deverá também executar o comando\n"
"    semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH' para adicionar "
"este\n"
"    novo caminho às predefinições do sistema. Se não pretendia usar\n"
"    o $TARGET_PATH como uma imagem xen, isto tanto poderá indicar um erro\n"
"    ou uma tentativa de intrusão.\n"
"    "

#: ../src/xen_image.py:41
msgid ""
"\n"
"    You can alter the file context by executing chcon -t xen_image_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"xen_image_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Pode alterar o contexto do ficheiro executando chcon -t xen_image_t "
"'$TARGET_PATH'\n"
"    Também precisa de alterar o contexto por omissão dos ficheiros no "
"sistema para os preservar mesmo na caso de uma etiquetagem total.  "
"\"semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH'\"\n"
"    "
